wordpress-mobile/WordPress-iOS
# Integrates Aztec MediaAttachment updates

## Discussion

This PR:
* Brings in the [media attachment updates](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/804) from Aztec
* Updates when the action sheet is activated in (now with a single tap)
* Changes the action text in the action sheet
* Adds a 3pt border to selected media attachments

Fixes #8063 

## To Test:
Preconditions: Build and run WPiOS, open the Aztec editor. Test on both iPads and iPhone devices.

* Verify the new dark grey bg overlay + wpblue border is visible when a media attachment is tapped
* Verify the no `message` text displays when a media attachment is selected — the only time text should display is when an upload fails.
* Check that tapping media attachments now brings up the action sheet immediately
* When the progress bar is displayed during an upload, the border should not display — verify this is the case.
* Turn off the network connection and try uploading something. After the upload errors out, verify that the media attachment displays error text in `message`. 
* Test both image and video media attachment types to make sure everything works as expected

Also, try opening an post with a lot of media attachments. Make sure selection works as expected.

@SergioEstevao would you mind putting some 👀 on this? Thanks!
